evalml/tests/model_understanding_tests/test_graphs.py

```diff
@@ -663,7 +663,7 @@ def test_jupyter_graph_check(import_check, jupyter_check, X_y_binary, X_y_regres
     jupyter_check.return_value = True
     with pytest.warns(None) as graph_valid:
         graph_partial_dependence(clf, X, features=0, grid_resolution=20)
-        assert len(graph_valid) == 1  # scikit-learn partial_dependence warning
+        assert len(graph_valid) == 1
```

Comment:
Indifferent about keeping this comment here so we know where this warning is coming from (aka not us) /shrug

Reply:
There was initially a warning being displayed I believe, by changing the `ARIMA` implementation to the non-deprecated one, it removed another warning. So it went from `len(graph_valid) == 2` to `len(graph_valid) == 1`